andela/Ah-backend-guardians
# #162685649 Upgrade Django version from 1.10 to 2.1

**What does this PR do?**
This PR delivers a chore that upgrades the Django version for Author's Haven from 1.10 to 2.1 which is the latest and has support for Python 3.x.

**Description of Task to be completed?**
Django's version 2.x is the latest release of Django and has full support for the Python 3.x series.

**Any background context you want to provide?**
With Django 1.x, URLs were declared using regular expressions which could be easily got wrong so the declaration of URLs in Django 2.x was changed to using path() which makes it easier.

**What are the relevant Pivotal Tracker stories?**
[#162685649](https://www.pivotaltracker.com/n/projects/2232496)

**How should be tested manually?**
Run each of the following commands in your terminal in the respective order.

- `git clone https://github.com/andela/Ah-backend-guardians.git`
- `cd Ah-backend-guardians`
- `git checkout ch-upgrade-162685649`
- `virtualenv venv`
- `. venv/bin/activate`
- `pip install -r requirements.txt`
- `pip freeze`

You should be able to get back the output as in the screenshot.

**Screenshots (if necessary)**
<img width="496" alt="screen shot 2019-01-08 at 10 29 44 am" src="https://user-images.githubusercontent.com/38134382/50816136-b2c31c80-1330-11e9-920c-bd6fb9812bea.png">
